Classes_Homework_5/modules/dbconnect.py

```diff
@@ -0,0 +1,38 @@
+import sqlite3 as sql
+
+from exec_utils.configloader import Config
+from modules.file import Files
+
+cnf = Config()
+f = Files()
+
+label1, label2, label3 = cnf.get_values("LABELS", "news_label"), \
+                         cnf.get_values("LABELS", "ad_label"), \
+                         cnf.get_values("LABELS", "recipe_label")
+
+
+class DBconnection:
+    def __init__(self, db_name):
+        with sql.connect(f.get_path(db_name)) as self.con:
+            self.curs = self.con.cursor()
+
+    def go(self, statement, predicates=''):
+        """Execute sql statement"""
+        self.curs.execute(statement, predicates)
+        self.con.commit()
+
+    def create_table(self, table_name):
+        if table_name == label1:  # News
+            self.go(cnf.get_values("SQL", "create_table_news"))
+        elif table_name == label2:  # Private ad
+            self.go(cnf.get_values("SQL", "create_table_ad"))
+        elif table_name == label3:  # Recipe
+            self.go(cnf.get_values("SQL", "create_table_recipe"))
```

Comment:
This code works well, but probably it will not look Pythonic if you have a lot of tables (for example, 200 tables -> 200 "if" statements).
If we change the names of attributes in the config.ini (create_table_news -> news, create_table_ad -> ad), we could write:
```
        try:
            self.go(cnf.get_values("SQL", table_name))
        except NoOptionError:
            print(f"SQL command for table {table_name} is not founded in config file")

```